Makefile

```diff
@@ -15,9 +15,10 @@
 ## program will usually tell you what to do.
 ##
 ## Once you have prepared your environment for running docker, you can
-## test the build by issuing 'make test', which at present is
-## synonymous with 'make html'. I.e., the website build is tested by
-## rendering all HTML content.
+## test the build by issuing 'USERID=0 GROUPID=0 make test', which at 
```

Comment:
Perhaps it would be better to include a whole section of docu-comment for different OSes?